src/components/Invitations/InviteProvider.test.tsx

```diff
@@ -0,0 +1,143 @@
+import React from 'react';
+import { act, renderHook } from '@testing-library/react-native';
+import { QueryClient, QueryClientProvider } from 'react-query';
+import { InviteProvider } from './InviteProvider';
+import { InviteParams, inviteNotifier } from './InviteNotifier';
+import { usePendingInvite } from '../../hooks/usePendingInvite';
+import { useAuth } from '../../hooks/useAuth';
+import { useActiveAccount } from '../../hooks/useActiveAccount';
+import { useAcceptInviteMutation } from '../../hooks/useAcceptInviteMutation';
+
+jest.mock('../../hooks/useAuth', () => ({
+  useAuth: jest.fn(),
+}));
+jest.mock('../../hooks/useAuth', () => ({
+  useAuth: jest.fn(),
+}));
+jest.mock('../../hooks/useActiveAccount', () => ({
+  useActiveAccount: jest.fn(),
+}));
+jest.mock('../../hooks/useAcceptInviteMutation', () => ({
+  useAcceptInviteMutation: jest.fn(),
+}));
+
+const useAuthMock = useAuth as jest.Mock;
+const useActiveAccountMock = useActiveAccount as jest.Mock;
+const useAcceptInviteMutationMock = useAcceptInviteMutation as jest.Mock;
+
+const refetch = jest.fn();
+const setActiveAccountId = jest.fn();
+const mutateAsync = jest.fn();
+const refreshForInviteAccept = jest.fn();
+const mockInviteParams: InviteParams = {
+  inviteId: 'invite-id',
+  evc: 'evc',
+};
+const mockAuthResult = {
+  accessToken: 'access-token',
+};
+
+const renderHookInContext = async () => {
+  return renderHook(() => usePendingInvite(), {
```

Comment:
You might checkout this helper @bruce-glazier added: https://github.com/lifeomic/react-native-sdk/blob/master/src/common/testHelpers/testing-library-wrapper.tsx

It should help with setup and request mocking.

Reply:
👍 I'll try that out next round